backend/migrate.ts

```
@@ -0,0 +1,624 @@
+        }
+
+        const discoveredPostIds = Object.keys(rootData).filter(key => {
+            const knownNonPostPrefixes = ['user:', 'email_to_id:'];
+            const knownExactNonPostKeys = ['users', 'user_ids', 'feedItems', 'allStoryTreeIds', 'imported_post_ids', 'postMetadata', 'replyMetadata', 'userMetadata', 'indexes', 'feedStats'];
+
+            if (knownExactNonPostKeys.includes(key) || knownNonPostPrefixes.some(prefix => key.startsWith(prefix))) {
+                logger.debug(`Key '${key}' at root is a known non-post key/prefix. Filtering out.`);
+                return false;
+            }
+            
+            if (rootData[key] && typeof rootData[key] === 'object' && rootData[key].hasOwnProperty('storyTree')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)